src/clj/game/cards/programs.clj

```diff
@@ -1,14 +1,319 @@
 (ns game.cards.programs
   (:require [game.core :refer :all]
             [game.utils :refer :all]
-            [game.macros :refer [effect req msg wait-for continue-ability]]
+            [game.macros :refer [effect req msg wait-for continue-ability when-let*]]
             [clojure.string :refer [split-lines split join lower-case includes? starts-with?]]
             [clojure.stacktrace :refer [print-stack-trace]]
             [jinteki.utils :refer [str->int other-side is-tagged? has-subtype?]]
             [jinteki.cards :refer [all-cards]]))
 
+(defmacro get-strength
```

Comment:
Why is this a macro and not a function?

Reply:
I honestly don’t remember why I made this a macro. It should probably be a plain function.